coderd/debug.go

```diff
@@ -23,6 +23,17 @@ func (api *API) debugCoordinator(rw http.ResponseWriter, r *http.Request) {
 	(*api.TailnetCoordinator.Load()).ServeHTTPDebug(rw, r)
 }
 
+// @Summary Debug Info Tailnet
+// @ID debug-info-wireguard-coordinator
+// @Security CoderSessionToken
+// @Produce text/html
+// @Tags Debug
+// @Success 200
+// @Router /debug/tailnet [get]
+func (api *API) debugTailnet(rw http.ResponseWriter, r *http.Request) {
+	api.agentProvider.ServeHTTPDebug(rw, r)
```

Comment:
This leaks lists of peers and IP addresses and stuff so it should be admin-only.